source/about/deployment_and_editions.rst

```diff
@@ -0,0 +1,164 @@
+=================================
```

Comment:
Do we want this new page surfaced up via the LHS?

Also, small nit: the title of this new page and its filename don't match - "deployments"

Reply:
Yes we want this to surface on the LHS.

Reply:
Fixed the file name.

Reply:
Great! See https://github.com/mattermost/docs/pull/5067#discussion_r720479767 for details on adding this topic to the LHS. Happy to help!

Reply:
Added.